ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 import contextlib
 import getpass
+from multiprocessing.sharedctypes import Value
```

Comment:
Where is this used?

Reply:
Nowhere, it must have been acted by my IDE.